charts/internal/calico/templates/node/felix-monitoring-service.yaml

```
@@ -0,0 +1,12 @@
+{{- if .Values.config.monitoring.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: felix-metrics-svc
```

Please, align the file name with the resource name. 
Either 
```suggestion
  name: felix-monitoring
```
Or rename the file to `felix-metrics-service.yaml`.